samples/Mvc.DynamicCorsPolicyProvider/Readme.md

```diff
@@ -0,0 +1,10 @@
+Mvc.DynamicCorsPolicyProvider Sample
+------------------------------------
+This sample demonstrates a way to dynamically load CORS policies in a running application without restarting it. 
+
+ICorsPolicyProvider is the DI injectable service through which MVC retrieves CORS policy for a request. The default CORS policy 
```

Comment:
BTW I don't think you need newlines here - GitHub will render it fine without newlines.